modular_zubbers/modules/customization/modules/language/_language_holder.dm

```diff
@@ -0,0 +1,30 @@
+/datum/language_holder/felinid
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
```

Comment:
```suggestion
/datum/language_holder/felinid/Initialize()
	. = ..()
	understood_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
	spoken_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
```

Doing something like this. Should make it compatible incase skyrat adds more languages to these.
Do mind this is just atop of my head and might not be entirely right, but it works as an example